cpp/include/cugraph/detail/decompress_matrix_partition.cuh

```diff
@@ -180,6 +180,217 @@ void decompress_matrix_partition_to_fill_edgelist_majors(
   }
 }
 
+template <typename vertex_t, typename edge_t, typename weight_t, typename prop_t, bool multi_gpu>
+__global__ void decompress_edgelist_high_degree(
```

Comment:
I have been naming this kind of kernels as "for_all_..." but I think this is a better name (and I have been worrying about potential name conflicts). So, I think we'd better set some naming conventions for this kind of kernels.

I think this name here is good except that `decompress_edgelist` suggests we're decompressing an edge list, but we're actually (partially) decompressing an edge partition to edge list.

Could you rename this function and `for_all..` functions in this file? (Then, I will rename other for_all functions in the primitives in a separate PR).

It depends on how we rename the caller function but one thought is to rename existing for_all_... functions to decompress_to_edgelist_high_degree (or mid_degree and so on) and this to something like partially_decompress_to_edgelist_high_degree?